ci-operator/step-registry/windows/e2e/operator/test/windows-e2e-operator-test-ref.yaml

```diff
@@ -6,6 +6,9 @@ ref:
     requests:
       cpu: 1000m
       memory: 400Mi
+  dependencies:
+    - env: WMCO_IMAGE
```

Comment:
Should this be `OPERATOR_IMAGE` instead?

Reply:
just curious to know what will happen if this variable is not set externally ? do we have any defensive check to handle that situation ?

Reply:
@selansen  ll add a defensive check in our hack script where we use it.